.github/workflows/push-apisix-base-image.yml

```diff
@@ -39,7 +39,16 @@ jobs:
         run: |
           docker build -t api7/apisix-base:${{ steps.tag_env.outputs.version }} \
             -f ./dockerfiles/Dockerfile.apisix-base.apk .
-      
+
+      - name: Run apisix-base docker
```

Comment:
Better to verify it in the build but not the upload. Please add it to https://github.com/api7/apisix-build-tools/blob/e3a5cc9abf66a82892b61582038e03a53d2d323d/.github/workflows/package-apisix-base-apk.yml#L30

Reply:
OK Thanks~